pkg/gitreceive/build.go

```diff
@@ -14,8 +14,13 @@ import (
 	"github.com/deis/builder/pkg"
 	"github.com/deis/builder/pkg/gitreceive/log"
 	"github.com/deis/builder/pkg/gitreceive/storage"
-	"github.com/pborman/uuid"
 	"gopkg.in/yaml.v2"
+
+	"k8s.io/kubernetes/pkg/api"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/watch"
```

Comment:
I don't see these dependencies explicitly defined in the `glide.yaml` file, and I believe that they're a transitive dependency from `github.com/deis/pkg`. That package uses version `~1.1` (https://github.com/deis/pkg/blob/master/glide.yaml#L4-L5), which is indeed our target for now, but I'd like to explicitly define the version in this repository in case the `deis/pkg` changes it in the future.